### ISSUE TITLE
please update modernizr

### ISSUE THREAD

**Issue by choiks141** [yeoman/generator-webapp_DEPRECATED#324]
modernizr is 0.5.2
and usage is changed

please update :)

**Comment by lhwparis** [comment]
grunt-modernizr was updated to 0.5.2 in trunk a few days ago. use the latest version from trunk. what do you mean with usage is changed?

**Comment by choiks141** [comment]
Gruntfile.js
modernizr: {
            devFile: 'bower_components/modernizr/modernizr.js',
            outputFile: '<%%= config.dist %>/scripts/vendor/modernizr.js',
            files: [
                '<%%= config.dist %>/scripts/{,_/}_.js',
                '<%%= config.dist %>/styles/{,_/}_.css',
                '!<%%= config.dist %>/scripts/vendor/*'
            ],
            uglify: true
        }

but config is changed.

modernizr: {

```
dist: {
    // [REQUIRED] Path to the build you're using for development.
    "devFile" : "lib/modernizr-dev.js",

    // [REQUIRED] Path to save out the built file.
    "outputFile" : "build/modernizr-custom.js",

    // When parseFiles = true, this task will crawl all *.js, *.css, *.scss files, except files that are in node_modules/.
    // You can override this by defining a "files" array below.
     "files" : {
        // "src": []
     }
}
```

}

https://github.com/Modernizr/grunt-modernizr#config-options

**Comment by sindresorhus** [comment]
Yes, this is a known issue: https://github.com/yeoman/generator-webapp/pull/318

tl;dr author of grunt-modernizr doesn't know semver.

**Comment by choiks141** [comment]
thanks. :+1:
